services/web/server/src/simcore_service_webserver/workspaces/_workspaces_db.py

```diff
@@ -125,6 +134,13 @@ async def list_workspaces_for_user(
         .where(workspaces.c.product_name == product_name)
     )
 
+    if filter_trashed is not None:
+        base_query = base_query.where(
+            workspaces.c.trashed.is_not(None)
+            if filter_trashed
```

Comment:
Is this right? `filter_trasher` is already not None here

Reply:
but it can be `False`/`True`

Reply:
I am still a bit confused .. so what is difference between None, True and False? 
- `None` you return everything
- Filter trashed = True you return (everything - trashed)
- Filter trashed = False you return just trashed
?